aci/resource_aci_cloudsubnet.go

```diff
@@ -135,16 +149,30 @@ func setCloudSubnetAttributes(cloudSubnet *models.CloudSubnet, d *schema.Resourc
 		d.Set("scope", scopeGet)
 	}
 	d.Set("usage", cloudSubnetMap["usage"])
+	d.Set("subnet_group_label", cloudSubnetMap["subnetGroup"])
 	return d, nil
 }
 
+func setCloudSubnetRelationalAttributes(cloudSubnetCont *container.Container, d *schema.ResourceData) {
+	ChildContList, err := cloudSubnetCont.S("imdata").Index(0).S(models.CloudsubnetClassName, "children").Children()
+	if err != nil {
+		log.Printf("[DEBUG]: Failed to set cloudSubnet relational attributes : %v", err)
```

Comment:
CloudSubnet -> Cloud Subnet

Reply:
It would help while debugging the error messages with the original class name.